internal/controller/api/management.go

```diff
@@ -203,8 +222,15 @@ func (s *ManagementServer) handleConnectionListing() http.HandlerFunc {
 
 	return func(w http.ResponseWriter, req *http.Request) {
 
+		principal, _ := middlewares.GetPrincipal(req.Context())
+		requestId := request_id.GetReqID(req.Context())
+		logger := logger.Log.WithFields(logrus.Fields{
+			"account":    principal.GetAccount(),
+			"request_id": requestId})
+
+		logger.Debugf("Getting connection list")
```

Comment:
Did you mean to add some variables into this message?

Reply:
I'm basically just logging that we received a request, who the request was made by (account field) and the request_id.